Add a readme to the npm package (#52)
fermyon/spin-js-sdk: types/README.md

## Type definitions for spin-js-sdk

> This package provides the type defintions for building apps with TypeScript using `spin-js-sdjk`.

Reviewer: ```suggestion
This package provides the type definitions for building apps with TypeScript using `spin-js-sdk`.
```

> ### Adding the pacakage to your project

Reviewer: ```suggestion
### Adding the package to your project
```

```
npm install --save-dev @fermyon/spin-sdk-types
```

> ### Importing the types into typescript

Reviewer: ```suggestion
### Importing the types into TypeScript
```
